built-in-policies/policyDefinitions/Kubernetes/ContainerProbesRequired.json

```diff
@@ -0,0 +1,74 @@
+{
+  "properties": {
+    "displayName": "Ensure all pods have readiness or liveness probes configured",
+    "policyType": "BuiltIn",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy enforces that all pods have a readiness and/or liveness probes configured. This policy is generally available for Kubernetes Service (AKS), and preview for AKS Engine and Azure Arc enabled Kubernetes. For instructions on using this policy, visit https://aka.ms/kubepolicydoc.",
+    "metadata": {
+      "version": "1.0.1",
+      "category": "Kubernetes"
+    },
+    "parameters": {
+      "requiredProbesList": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Required probes list",
+          "description": "The list of probes readinessProbe and/or livenessProbe required for each pod."
```

Comment:
Please use `"The list of probes that are required to be defined on a container. Kubernetes currently supports 'livenessProbe', 'readinessProbe', and 'startupProbe'."`

Reply:
Ok, fixed it.